drivers/video/isx012.c

```diff
@@ -635,6 +637,16 @@ static struct imgsensor_ops_s g_isx012_ops =
  * Private Functions
  ****************************************************************************/
 
+static void i2c_lock(void)
```

Comment:
i2c may be shared by multiple device, is it enough to lock from device driver?

Reply:
For this case I think that it is enough. What are your concerns?